4.1/crud-operation-fetch.md

```diff
@@ -64,6 +64,29 @@ class ProductCrudController extends CrudController
 
 Based on the fact that the ```fetchTag()``` method exist, the Fetch operation will create a ```/product/fetch/tag``` POST route, which points to ```fetchTag()```. Inside ```fetchTag()``` we call ```fetch()```, that responds with entries in the format ```select2``` needs.
 
+**Preventing FetchOperation of guessing the searchable attributes**
+
+If not specified `searchable_attributes` will be automatically infered from model dabatase columns. To do your own search and replace default FetchOperation behaviour you can setup an empty `searchable_attributes`.
```

Comment:
@pxpm I'm afraid I don't understand 😅 Search&replace? What replace?

Could you please rephrase in more words so I understand when this is needed? Happy to help make this easier to understand 😀